docs/plugin-development.rst

```diff
@@ -30,7 +30,7 @@ our example, we'll just use JSON. ::
     touch cats/plugin.json
 
 The plugin config file should specify a human-readable name and description for your
-plugin, and can optionally contain a list of other plugins that your plugin
+plugin, and can optionally contains a URL to a documentation, a list of other plugins that your plugin
```

Comment:
This paragraph should be improved. One suggestion:

> "The plugin config file should specify a human-readable name and description for your plugin. It can also optionally contain a URL to documentation and a list of other plugins that your plugin depends on."